src/App.tsx

```diff
@@ -45,14 +46,13 @@ function App() {
             setRadiusKm={setRadiusKm}
             lastUpdateTime={lastUpdateTime}
           />
-
-          <WalkInSection coords={coords} radiusKm={radiusKm} />
-
           <BookingSection
             coords={coords}
             radiusKm={radiusKm}
             setLastUpdateTime={setLastUpdateTime}
           />
+          <WalkInSection coords={coords} radiusKm={radiusKm} />
```

Comment:
Do we want these underneath the calendar? As the calendar is much larger than walk-in and crowd sourced locations I can foresee that many people wouldn't scroll far enough to know that these sections are there.